File: CoreJavaDay2Assignment/Student.java
package com.sonata;

public class Student {
	int stdID;
	String stdName;
	int stdClass;
	//constructor
	Student()
	{
	}
	//parameterized constructor
	Student(int id,String name,int stdclass)
	{
		this.stdID=id;
		this.stdName=name;
		this.stdClass=stdclass;
	}
	//method display
	public void display()
	{
		System.out.println(stdID);
		System.out.println(stdName);
		System.out.println(stdClass);
	}
	public static void main(String args[])
	{
		Student s=new Student();//object
		s.stdID=1;
		s.stdName="varun";
		s.stdClass=9;
		s.display();//method calling
		
		Student s1=new Student(2,"veera",10);
		s1.display();
	}

}

File: Typescript_Assignment/Student.js
var Student = /** @class */ (function () {
    function Student(id, name, classname) {
        this.stdID = id;
        this.stdName = name;
        this.stdClass = classname;
    }
    Student.prototype.display = function () {
        console.log(this.stdID);
        console.log(this.stdName);
        console.log(this.stdClass);
    };
    return Student;
}());
var student1 = new Student(12, "veera", "Cse");
student1.display();
var student2 = new Student(21, "venkat", "ece");
student2.display();
